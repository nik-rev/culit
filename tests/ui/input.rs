//! No arguments can be passed to the proc macro

use culit::culit;

mod custom_literal {
    pub mod integer {}
}

#[culit(no input allowed)]
fn main() {}
