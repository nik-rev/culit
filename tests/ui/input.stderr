error: expected `#[culit(local)]` or `#[culit]`
 --> tests/ui/input.rs:9:9
  |
9 | #[culit(no input allowed)]
  |         ^^

error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/ui/input.rs:10:13
   |
10 | fn main() {}
   |             ^ consider adding a `main` function to `$DIR/tests/ui/input.rs`
